site/src/pages/components/popover-hint.research.explainer.mdx

```diff
@@ -37,120 +34,94 @@ A followup proposal to the [original Popover proposal](https://open-ui.org/compo
 
-The motivating use case, but certainly not the only use case, for these two capabilities is in the construction of "tooltips". Typically, tooltips are transient bits of additional information related to a bit of content. This additional information is *not required* and is supplementary in nature. Usually the user accesses this tooltip by hovering, keyboard-focusing, or long-pressing a control.
+It turns out, the definition of "tooltip" can be tricky. There are several definitions people tend to use:
 
-# "Hover" triggering
+1. The "original" definition: a **text-only "tip"** about a "tool" button. For example, "Insert link" for this icon: ![insert link icon](/images/insert-link-button.png). The web platform has a [fraught](https://www.24a11y.com/2017/the-trials-and-tribulations-of-the-title-attribute/) implementation of this definition, via the `title` attribute.
+2. The "triggered by hover" definition: any **pop-up window/dialog**, containing **any type of content**, that **opens when a user mouse-hovers** an element.
+3. The "semantic" definition: A popover containing **non-essential**, **auxiliary** information about a context element.
 
-The existing popover API includes the concept of "invoking elements":
+Definition <span>#</span>1 is too limiting, and does not capture common use cases that include non-textual content. A good example is the window that appears when you hover over a username on Github:
 
-```html
-<button popovertarget=foo popovertargetaction=show>
-  Click me to show a popover
-</button>
-<div popover id=foo> I'm a popover </div>
-```
+  ![Github user information tooltip](/images/github-tooltip.png)
 
-The `popovertarget` attribute can be used on buttons (and `<input type=button>`), and it points to the idref of an element with the `popover` attribute. In this configuration, clicking the button automatically shows the popover, without requiring Javascript. In addition, this linkage ensures that the accessibility tree is properly updated to represent the connection. The `popovertargetaction` attribute determines what action to take when the button is activated: `show`, `hide`, or `toggle`.
+Definition <span>#</span>2 is mis-guided. Many things can be "hover-triggered" without being tooltips. A great example is a hover-triggered menu, where hovering over a sub-menu item brings up the sub-menu. Most people agree that the sub-menu is **not** a tooltip. It just happens to be hover-triggered.
 
-## Add the `interest` value
+That leaves definition <span>#</span>3, which I believe is the correct definition. A tooltip should not contain directly-necessary information - that should be placed into the page content directly. It should provide auxiliary or extra information that is available only when the user wants to dig deeper into some content. And that extra information should be able to contain more than just text, since the auxiliary information might very well contain or even **be** an image or other non-text content.
```

Comment:
I'd recommend removing `I` from this statement. Possibly something like, "which is the recommended definition". You should probably open an issue to get alignment on the definition and upon resolution make this *the* definition and move the other considerations to an appendix.

Reply:
Yeah good point. I changed it to roughly your suggestion, plus some links.